paper/sections/materials.tex

```diff
@@ -51,14 +51,17 @@ \subsection{Dataset}
 percentage of all standard amino acids present within a protein, as defined by~\cite{gromiha2010protein}. Also known
 as Monopeptide Composition (MPC).
 \item \textbf{Dipeptide Composition (DPC)}: a feature vector of $400$ values ranging from $0$ -- $100$ indicating the
-percentage of all possible amino acid pairs present within a protein, as defined by~\cite{gromiha2010protein}.
+percentage of all possible ordered amino acid pairs present within a protein, as defined by~\cite{gromiha2010protein}.
 \item \textbf{Physico-Chemical Composition (PHC)}: a feature vector of $11$ values corresponding
 to percentage composition of physico-chemical residue classes, including: Aliphatic, Neutral, Aromatic, Hydrophobic, Charged, Positively charged,
 Negatively charged, Polar, Small, Large, and Tiny. 
 \item \textbf{Biochemical Composition (AAindex)}: a feature vector of $49$ physical, chemical, energetic, and
 conformational amino acid properties which have been averaged across all amino acids present within the protein.
-\item \textbf{Position-specific scoring matrix (PSSM) profile}: a feature vector of $400$ min-maxed scaled values 
-being generated by summing up all the rows (scores) in the PSSM profile of a sequence that correspond to the same amino acid.
+\item \textbf{Position-specific scoring matrix (PSSM) profile}: A Position-Specific Scoring Matrix (PSSM), 
+is a scoring matrix in which amino acid substitution scores are given separately for each position in 
+a protein multiple sequence alignment. This feature is a min-maxed scaled vector of $400$ values being 
+generated by summing up all the rows of the PSSM of a sequence corresponding to the same amino acid. 
+The PSSM profile was generated by running PSI-BLAST against SwissProt database with the BLOSUM62 matrix.
```

Comment:
```suggestion
\item \textbf{Position-specific scoring matrix (PSSM) profile}: a feature vector of $400$ sequence
likelihoods aggregated across min-max scaled probabilities for each ordered amino acid pair within
proteins in the SwissProt database.
```

Reply:
@HamidHeyde this is the other change to look at right now - I think you'll just need to hit "approve" and we're all set

Reply:
@gkiar: You guys are more experienced in this regard, if you think these 2 are equal, please go ahead and change it, cause i couldn't make it shorter than those 3 sentences.